static/js/biomedical/disease/chart.ts

```diff
@@ -63,6 +63,18 @@ export interface CompoundDiseaseTreatmentData {
   // FDA clinical phase for which the compound has been studied
   clinicalPhaseNumber: number;
 }
+
+export interface CompoundDiseaseContraindicationData {
```

Comment:
make the types.ts file and put this there. Can defer to next PR, but please make that PR next.

Reply:
I'll definitely do it on the next PR!